wgsl/index.bs

```diff
@@ -11287,6 +11302,9 @@ For a given group of invocations:
 The remaining subsections specify a static analysis that verifies that
 [[#collective-operations|collective operations]] are only executed in [=uniform
 control flow=].
+If the [=language_extension/subgroup_uniformity=] feature is supported then
+there are multiple scopes of uniformity.
+This analysis is performed once for each scope.
```

Comment:
Perhaps this is too subtle, but it is much simpler to describe in the spec as simply executing the analysis twice (once per scope).